Update golang sample to assume the okteto manifest is already created… (#119)
okteto/docs: src/content/samples/golang.mdx

## Prerequisites

> - Install the Okteto CLI >= 2.0.0. Follow this [guide](/docs/cloud/okteto-cli/) if you haven't done it yet.

Reviewer: I don't think is necessary to specify the version. Maybe just mention 'install the latest version of the Okteto CLI'?

$ cd go-getting-started
```

> The `okteto.yml` file describes how to [build](/docs/reference/manifest/#build-object-optional) and [deploy](/docs/reference/manifest/#deploy-string-optional) the Go Sample App.

Reviewer 2: ```suggestion
At the root of the directory, you'll find the `okteto.yaml` file. This describes how to [build](/docs/reference/manifest/#build-object-optional) and [deploy](/docs/reference/manifest/#deploy-string-optional) the Go Sample App.
```

Did you notice that you're accessing your application through an HTTPs endpoint? This is because Okteto Cloud will [automatically create them](/docs/cloud/ssl/) for you when you deploy your application. Cool no 😎?

## Step 2: Activate your development container

The [dev](http://localhost:8080/docs/reference/manifest/#dev-object-optional) section defines how to activate a development container for the Go Sample App:

```yaml
dev:
  hello-world:
    image: okteto/golang:1
    command: bash
    sync:
      - .:/usr/src/app
    volumes:
      - /go
      - /root/.cache
    securityContext:
      capabilities:
        add:
          - SYS_PTRACE
    forward:
      - 2345:2345
```

The `hello-world` key matches the name of the hello world Deployment. The meaning of the rest of fields is:

- `image`: the image used by the development container. More information on development images [here](/docs/reference/development-environments/).
- `command`: the start command of the development container.
- `sync`: the folders that will be synchronized between your local machine and the development container.

> - `volumes`: a list of paths in your development container to be mounted as persistent volumes. For example, this is useful to persist the Go cache.

Reviewer 2: ```suggestion
- `volumes`: a list of paths in your development container to be mounted as persistent volumes. For example, this can be used to persist the Go cache.
```
